src/Modal/Modal.jsx

```diff
@@ -40,16 +65,48 @@ class Modal extends Component {
       >
         <ModalStyles>
           <Overlay onClick={onClose} />
-          <Content>
-            <Row textAlign="right">
-              <Col offset={11} lg={1}>
-                <CloseButton ref={this.onClose} onClick={onClose}>
-                  <Icon name="close" />
-                </CloseButton>
+          <Height>
+            <Row position="center" verticalAlign="middle">
+              <Col>
+                <Content>
+                  <Row textAlign="right" position="end">
+                    <Col lg={1}>
+                      <CloseButton ref={this.closeButton} onClick={onClose}>
+                        <Icon name="close" />
+                      </CloseButton>
+                    </Col>
+                  </Row>
+                  <BodyContent>
+                    {children}
+                    <ButtonsArea>
+                      <Row position="end">
+                        {onCancel && (
+                          <Col lg={0}>
+                            <Button type="white" onClick={onCancel}>
+                              {cancelText}
+                            </Button>
+                          </Col>
+                        )}
+                        {onSubmit && (
+                          <Col lg={0}>
+                            <Col lg={0}>
+                              <Button
+                                style={{ marginLeft: 16 }}
```

Comment:
Should all styles belong in the styles file?

Reply:
sometimes i get lazy.. is this really very bad here?